run/local-cluster.sh

```diff
@@ -95,23 +96,29 @@ echo "* ${DNSMASQ_NAME}"
 # Local ports.
 PORT=8080
 
+# Start a data volume node for shared data.
+CERTS_DIR="/certs"
+CERTS_NAME="cockroach_certs"
+# Generate certs.
+docker run -v ${CERTS_DIR} --name=${CERTS_NAME} ${COCKROACH_IMAGE} create-ca-cert --certs=${CERTS_DIR} 2> /dev/null
+docker run --volumes-from=${CERTS_NAME} ${COCKROACH_IMAGE} create-node-cert --certs=${CERTS_DIR} localhost 127.0.0.1 2> /dev/null
```

Comment:
I think you might be able to add a `--rm` to this container to minimize leftovers.